questionnaire/2020/questions.yaml

```diff
@@ -215,6 +215,20 @@
     - A little
     - A lot
 
+  - title: What connection methods / transports are being utilized as part of your network automation implementation?
+    type: Multiple choice 
+    id: transport-protocol
+    responses:
+    - CLI
+    - Netconf
+    - Restconf
+    - gNMI
+    - gRPC
```

Comment:
Do we want to save gRPC for "data sources are you using to detect problems in your network" instead of mentioning it here?

Reply:
gRPC is starting to get rolled out with some vendors (such as Nokia) for automation connectivity AFAIK

Reply:
For making config changes? Not just for dial-out telemetry?

Reply:
yup, config changes, see: https://github.com/ansible/ansible/pull/67472 but not yet in Ansible, but it's definitely a "thing" to track for the future I'd assume (the move to NETCONF and gRPC I'd say)

Reply:
Yes you can use gNMI (based on gRPC) to change the configuration, most vendors support it today